nest_elephant_tvb/translation/mpi_translator.py

```diff
@@ -0,0 +1,135 @@
+import time
+import numpy as np
+from mpi4py import MPI
+from threading import Thread, Lock
+from nest_elephant_tvb.translation.science_nest_to_tvb import store_data,analyse_data
+
+lock_status=Lock() # locker for manage the transfer of data from thread
+
+def init(path, param, comm_receiver, comm_sender, logger_master, logger_receive, logger_send):
+    
+    store = store_data(path+'/log/',param)
+    analyse = analyse_data(path+'/log/',param)
+    # variable for communication between thread
+    status_data=[0] # status of the buffer
+    initialisation =np.load(param['init']) # initialisation value
+    buffer=[initialisation] # buffer of the rate to send it
+        
+    # create the thread for receive and send data
+    th_receive = Thread(target=receive, args=(logger_receive,store,status_data,buffer, comm_receiver))
+    th_send = Thread(target=send, args=(logger_send,analyse,status_data,buffer, comm_sender))
+
+    # start the threads
+    # FAT END POINT
+    logger_master.info('Start thread')
+    th_receive.start()
+    th_send.start()
+    th_receive.join()
+    th_send.join()
+    logger_master.info('thread join')
+    return
+    
+    
+def receive(logger,store,status_data,buffer, comm):
```

Comment:
Comm variable is not documented in the param list

Reply:
Old code from the threaded version. Will be changed soon (the next commits).